docs/src/operations/validator-initiatives.md

```diff
@@ -13,12 +13,15 @@ The Solana Foundation helps facilitate the growth of the consensus validator net
 
 ## Tour De Sun 22
 
-Delegation program participants who operate a performant testnet node, may also join Tour De Sun 22 (TdS22).  The program gives locked SOL incentive payments to operators for running approved hardware on Testnet. For more information, see [TdS22](https://solana.org/tds22).
+Tour De Sun 22 is winding down. Applications to the program are now closed.
+March 2024 was the final month of the program. The program is closed and will not accept new applicants.
+
+Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.
```

Comment:
@tigarcia - Any thoughts on this? I don't necessarily like duplicating the same content from this page:
https://solana.org/tds22

But, I also see the point about not having outdated information. Some options come to mind:
- Remove this section from this doc altogether
- Put a very brief note here to serve as a tombstone, something like below

> The Tour de Sun 22 (TdS22) has wound down, see [TdS22](https://solana.org/tds22) for more information

Reply:
+

Reply:
I think we should go even briefer:

> The Tour De Sun 22 is closed and no longer accepting applications. Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.